### PR TITLE
Refactor cleanup stored mail job spec

### PR DESCRIPTION
**Pull request by Matthijsy**
### Summary
The cleanup stored mail job spec fails a lot. However, on my local it works all the times. I did a little refactor, hopefully this will fix the issues on the CI

### CODE REVIEW

**Comment by cmitz** [comment]
Haha too bad it fails. Thanks for picking this up though, I didn't have the time yet.

Maybe use Timecop?

**Comment by Matthijsy** [comment]
Why should we use Timecop? I don't think there is an timing issue

**Comment by cmitz** [comment]
Oh I think maybe it is. But also maybe the original code is a little borked, because in Slack I never see expired emails being soft-deleted, only really deleted...

File: spec/jobs/cleanup_expired_stored_mails_job_spec.rb [hunk 1]
```diff
@@ -2,58 +2,41 @@
...
       it { expect(job).to have_received(:inform_slack).with(0, 0) }
     end
 
     context 'when with one expired email and no deleted emails' do
-      before do
-        FactoryBot.create(:stored_mail, :expired)
-      end
-
-      it { expect { job.perform }.to change { StoredMail.only_deleted.count }.from(0).to(1) }
+      let(:stored_mail) { FactoryBot.create(:stored_mail, :expired) }
```
Comment (cmitz) [inline spec/jobs/cleanup_expired_stored_mails_job_spec.rb]: Vanwege lazy evaluation wordt deze stored_mail volgens mij nooit aangemaakt

**Pull request merged**